supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/api/ActionBuilder.scala

```diff
@@ -18,30 +19,32 @@ import scala.concurrent.duration._
   * Utility singleton to create high level actions.
   */
 object ActionBuilder {
-  object Labels {
-    object Key {
-      /**
-        * Very short description of the action
-        */
-      val Tag = "tag"
-      val Logging = "logging"
-      val InputName = "inputName"
-      val OutputName = "outputName"
-    }
-    object Value {
-      val ContainerSetup = "ContainerSetup"
-      val UserAction = "UserAction"
-      val Localization = "Localization"
-      val Delocalization = "Delocalization"
-      val Background = "Background"
-      val RetryWithMoreMemory = "CheckingForMemoryRetry"
-    }
-  }
-
   implicit class EnhancedAction(val action: Action) extends AnyVal {
     private def javaFlags(flags: List[ActionFlag]) = flags.map(_.toString).asJava
 
+    /**
+      * Only for use with docker images KNOWN to not have entrypoints already set,
+      * or used with accompanying call to setEntrypoint("non-empty-string").
+      *
+      * Otherwise use withEntrypointCommand() below until the google issues listed in BA-6406 are fixed.
```

Comment:
https://broadworkbench.atlassian.net/browse/BA-6406 is "closed" as far as Jira is concerned and Google considers this "Done" too since we have a workaround. So we could probably remove the "until ..." part of this comment.

Reply:
Take a look at the new comment. Happy to tweak again.